### ISSUE TITLE
Remove Largo Sidebar/Footer Featured Posts widget *files*

### ISSUE THREAD

**Issue by benlk** [INN/Largo#897]
Now that the migration is set up in #895 for #881, we need to:

- [x] wait a release or two to be assured that everyone will have migrated successfully
- [x] remove the files for these widgets from the theme:
    - Largo Sidebar Featured Posts
    - Largo Footer Featured Posts
    - Largo Featured Posts

**Comment by benlk** [comment]
Removing these files at the same time as #895 would trip Wordpress' deactivated-widget safeties and remove the widgets from the sidebars on the first page load post-upgrade. There would be no widgets to migrate with the upgrade functions in #895.

This is why this is a separate ticket.

**Comment by aschweigert** [comment]
so when do we do this? as a hotfix immediately after 0.5.3 is deployed and the update script has run?

**Comment by benlk** [comment]
Bumping this to 0.5.5 until after #987 is deployed.

**Comment by benlk** [comment]
Yes, the Featured Posts widget should be removed as well, because of #987.
